src/common/transformations/src/transformations/common_optimizations/mark_precision_sensitive_subgraphs.cpp

```diff
@@ -32,10 +32,10 @@ bool ov::pass::MarkPrecisionSensitiveSubgraphs::run_on_model(const std::shared_p
         visited.insert(r.get());
     }
 
-    auto markup_func = [](Node* node) {
-        if (ov::is_type<ov::opset8::Constant>(node)) {
-            ov::disable_fp16_compression(node->shared_from_this());
-        }
+    auto markup_func = [this](Node* node) {
+        if (m_mark_only_consts && !ov::is_type<ov::opset8::Constant>(node))
```

Comment:
Now we seem to change the behaviour for other plugins (not only GPU) because they have `false` value for `m_mark_only_consts` so this lambda look like for them:
```
    auto markup_func = [](Node* node) {
        if (ov::is_type<ov::opset8::Constant>(node)) {
            ov::disable_fp16_compression(node->shared_from_this());
        }
```

but this transformation shold change nothing.
To me it should like this:

```
    auto markup_func = [](Node* node) {
        if(!m_mark_only_consts)
            return;
        if (ov::is_type<ov::opset8::Constant>(node)) {
            ov::disable_fp16_compression(node->shared_from_this());
        }
```

Reply:
as I can see all other plugins have `true` by default `explicit MarkPrecisionSensitiveSubgraphs(bool mark_only_consts = true) :`
`m_mark_only_consts = true`, in this case we should call `disable_fp16_compression` only for Const otherwise `return`
`m_mark_only_consts=false`, we should call `disable_fp16_compression`  for all nodes
the current impl looks correct

Reply:
Agree that other plugins should not be affected